<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>607</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>607</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="sysrootLabel">
     <property name="toolTip">
      <string>Path to the sysroot. Leave empty to use the local machine.</string>
     </property>
     <property name="text">
      <string>Sysroot:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditSysroot</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KUrlRequester" name="lineEditSysroot">
     <property name="toolTip">
      <string>Path to the sysroot. Leave empty to use the local machine.</string>
     </property>
     <property name="mode">
      <set>KFile::Directory|KFile::ExistingOnly|KFile::LocalOnly</set>
     </property>
     <property name="placeholderText">
      <string>local machine</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="applicationPathLabel">
     <property name="toolTip">
      <string>Path to the application binary and library.</string>
     </property>
     <property name="text">
      <string>Application Path:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditApplicationPath</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KUrlRequester" name="lineEditApplicationPath">
     <property name="toolTip">
      <string>Path to the application binary and library.</string>
     </property>
     <property name="mode">
      <set>KFile::Directory|KFile::ExistingOnly|KFile::LocalOnly</set>
     </property>
     <property name="placeholderText">
      <string>auto-detect</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="extraLibraryPathsLabel">
     <property name="toolTip">
      <string>List of colon-separated paths that contain additional libraries.</string>
     </property>
     <property name="text">
      <string>Extra Library Paths:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditExtraLibraryPaths</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KUrlRequester" name="lineEditExtraLibraryPaths">
     <property name="toolTip">
      <string>List of colon-separated paths that contain additional libraries.</string>
     </property>
     <property name="mode">
      <set>KFile::Directory|KFile::ExistingOnly|KFile::LocalOnly</set>
     </property>
     <property name="placeholderText">
      <string>empty</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="debugPathsLabel">
     <property name="toolTip">
      <string>List of colon-separated paths that contain debug information.</string>
     </property>
     <property name="text">
      <string>Debug Paths:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditDebugPaths</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KUrlRequester" name="lineEditDebugPaths">
     <property name="toolTip">
      <string>List of colon-separated paths that contain debug information.</string>
     </property>
     <property name="mode">
      <set>KFile::Directory|KFile::ExistingOnly|KFile::LocalOnly</set>
     </property>
     <property name="placeholderText">
      <string>auto-detect</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="kallsymsLabel">
     <property name="toolTip">
      <string>Path to the kernel symbol mapping.</string>
     </property>
     <property name="text">
      <string>Kallsyms:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditKallsyms</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KUrlRequester" name="lineEditKallsyms">
     <property name="toolTip">
      <string>Path to the kernel symbol mapping.</string>
     </property>
     <property name="mode">
      <set>KFile::Directory|KFile::ExistingOnly|KFile::LocalOnly</set>
     </property>
     <property name="placeholderText">
      <string>auto-detect</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="archLabel">
     <property name="toolTip">
      <string>System architecture, e.g. x86_64, arm, aarch64 etc.</string>
     </property>
     <property name="text">
      <string>Architecture:</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxArchitecture</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="comboBoxArchitecture">
     <property name="toolTip">
      <string>System architecture, e.g. x86_64, arm, aarch64 etc.</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>auto-detect</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x64</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ARMv7 (32 bit)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ARMv8 (64 bit)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="objdumpLabel">
     <property name="toolTip">
      <string>&lt;qt&gt;The &lt;tt&gt;objdump&lt;/tt&gt; binary that will be used for the disassembly view. Leave empty to let it be auto-detected.&lt;/qt&gt;</string>
     </property>
     <property name="text">
      <string>Objdump:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditObjdump</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="KUrlRequester" name="lineEditObjdump">
     <property name="toolTip">
      <string>&lt;qt&gt;The &lt;tt&gt;objdump&lt;/tt&gt; binary that will be used for the disassembly view. Leave empty to let it be auto-detected.&lt;/qt&gt;</string>
     </property>
     <property name="placeholderText">
      <string>auto-detect</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
